app/frontend/schools/courses/components/CourseAuthors__Root.res

```diff
@@ -135,9 +135,9 @@ let make = (~courseId, ~authors) => {
             Some(
               state.authors |> ArrayUtils.unsafeFind(
                 author => author |> Author.id == authorId,
-                "Could not find author with ID " ++
+                t("author_not_found") ++
                 (authorId ++
-                (" in the list of known authors for course with ID " ++ courseId)),
+                (t("in_the_list") ++ courseId)),
```

Comment:
You can merge it into a single entry and make `authroId` and `courseId` variables; 

```res
t("author_not_found", ~variables=[("authorId", authorId),("courseId", courseId)])
```

```yml
author_not_found: Could not find author with ID %{authorId} in the list of known authors for course with ID %{courseId}
```